content/press/press.bib

```diff
@@ -0,0 +1,107 @@
+@misc{cep_eager,
```

Comment:
I think maybe you forgot to add logic to pull in this press bib file and display it.

Reply:
That's for the next PR, haven't made a press page yet or merged the current press on the website with this file. I just made it as preparation

Reply:
Could also pull that file out of this commit for cleanliness

Reply:
Yes, can you do that please so we know what's going on with each PR?